lib/src/data/repository/movie_repo.dart

```diff
@@ -1,20 +1,20 @@
 import 'dart:convert';
 import 'package:flutter/services.dart';
-import '../utils/strings.dart';
-import '../model/movie_data.dart';
-import './interface/movie_interface.dart';
+import '../../core/utils/strings.dart';
+import '../../domain/entity/movie_data.dart';
+import '../../domain/repository/interface/movie_interface.dart';
 
 class MovieRepository extends MovieInterface {
   List<Movie> parsedMovieList = [];
 
-  Future<List<Movie>> parseMovies() async {
+  static Future<List<Movie>> parseMovies() async {
```

Comment:
You don't need this method to be static. Also I'd mark this as private

Reply:
Done.